themes/default/content/what-is/what-is-configuration-management.md

```diff
@@ -0,0 +1,73 @@
+* **Reliability**: Inconsistency is the bane of reliability. _Snowflake servers_ that have manual tweaks, incorrect hotfixes, or missing software updates make it harder to fix issues. Automation helps reduce inconsistency by eliminating human error and variance in manually-configured systems.
+* **Scalability:** Defined configurations that can be deployed via configuration management tool make it possible to scale systems faster, easier, and more reliably.
+* **Accountability**: Similar to infrastructure as code, configuration management enables the desired state to be checked into version control. Changes to the desired state can be tracked, providing accountability and visibility.
+
+## What are some practical use cases for configuration management?
+
+Configuration management has a number of practical use cases for many organizations:
+
+* Automated configuration of new systems (typically servers, but could also apply to network devices or other systems)
+* Recovery from outages
+* Remediation of security flaws
+* Patching or updating software or operating systems
+* Deployment of software packages or applications
+
+## Are there any disadvantages to configuration management?
+
+Like any practice, there are some potential disadvantages to configuration management. In most instances, these potential disadvantages can be outweighed by the benefits that configuration management brings:
+
+* **Learning curve**: Some configuration management tools may have a steep learning curve, requiring time and effort for teams to become proficient. This is especially the case for tools that leverage a DSL.
+* **Complexity**: Managing configurations for complex systems can be intricate, leading to potential challenges in defining and maintaining configurations accurately.
+* **Overhead**: Implementing configuration management may introduce some overhead, especially in the initial setup and learning phases.
+
+## How does configuration management relate to infrastructure as code?
+
+Although configuration management (as a process) can apply to managing cloud infrastructure, the implementation of most configuration management tools make them less suitable for use in an infrastructure as code use case than a dedicated infrastructure as code tool like Pulumi. However, configuration management tools are great companions to an infrastructure as code tool like Pulumi, offering the ability to perform fine-grained configuration changes on cloud infrastructure provisioned via Pulumi (for example, ensuring that the operating system on a cloud instance is configured in a specific way or with a specific software package or application).
```

Comment:
Should we link to an example of this in action?

Reply:
Good idea, done!